src/version.props

```diff
@@ -1,5 +1,5 @@
 <Project>
     <PropertyGroup>
-        <Version>1.1.3</Version>
```

Comment:
This is just for curiosity it doesn't really matter, but when it is a new feature should we add a 1.2.0 instead?

Reply:
I honestly didn't think about that at all - I am just so used to bumping the patch version 😂

I will think more about it next time ... pinkie promise

Reply:
🌟 Promise